cluster/gce/util.sh

```diff
@@ -444,43 +506,48 @@ function kube-up {
   # Wait for last batch of jobs.
   wait-for-jobs
 
-  # Create the routes, 10 at a time.
-  for (( i=0; i<${#MINION_NAMES[@]}; i++)); do
-    create-route "${MINION_NAMES[$i]}" "${MINION_IP_RANGES[$i]}" &
-
-    if [ $i -ne 0 ] && [ $((i%10)) -eq 0 ]; then
-      echo Waiting for a batch of routes at $i...
-      wait-for-jobs
-    fi
-
-  done
-  # Wait for last batch of jobs.
-  wait-for-jobs
-
   local -a scope_flags=()
   if (( "${#MINION_SCOPES[@]}" > 0 )); then
     scope_flags=("--scopes" "${MINION_SCOPES[@]}")
   else
     scope_flags=("--no-scopes")
   fi
-  # Create the instances, 5 at a time.
-  for (( i=0; i<${#MINION_NAMES[@]}; i++)); do
-    (
+  local scopes_flag="${scope_flags[@]}"
```

Comment:
The proper solution here would be to pass scope_flags as the last argument, shift the previous arguments and then just use `$@` to refer to the flags.

But considering we're butchering the spaces here already, I think the least I can expect here is using `*` instead of `@` since that makes the intention clear.

```shell
  local scopes_flag="${scope_flags[*]}"
```

Or just bypass defining the local variable and just use `"${scope_flags[*]}"` straight in the create-node-template call.